octobot_commons/dict_util.py

```diff
@@ -34,7 +34,7 @@ def find_nested_value(dict_, field):
 
 
 def get_value_or_default(dictionary, key, default=None):
-    if key in dictionary:
+    if dictionary and key in dictionary:
```

Comment:
I think a `if dictionary is not None and ...` is sufficient.

Reply:
right